libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -308,14 +308,14 @@ export class CarouselComponent implements OnInit, AfterContentInit, AfterViewIni
     @HostListener('keydown.arrowright', ['$event'])
     onKeydownArrowRight(event: KeyboardEvent): void {
         event.preventDefault();
-        this.next();
+        this.dir === 'rtl' ? this.previous() : this.next();
```

Comment:
This should be using the rtl service from core. We have a service that unifies all components to be RTL

Reply:
@stefanoScalzo @InnaAtanasova as I understand you pointed this service – https://github.com/SAP/fundamental-ngx/blob/master/libs/core/src/lib/utils/services/carousel.service.ts
`this.previous()` and `this.next()` methods using this service ([in next](https://github.com/SAP/fundamental-ngx/blob/5de5d4c890d2e53e7f74b4f4682a1cc32e56c1bd/libs/core/src/lib/carousel/carousel.component.ts#L356), [in previous](https://github.com/SAP/fundamental-ngx/blob/5de5d4c890d2e53e7f74b4f4682a1cc32e56c1bd/libs/core/src/lib/carousel/carousel.component.ts#L329)), but it works not in a proper way for RTL direction, and my changes are the fastest solution for this issue

Should I try to fix carousel.service.ts?